duckduckgo/iOS
# Fix Russian fireproofing translations

Task/Issue URL: https://app.asana.com/0/0/1202101260086320/f
Tech Design URL:
CC:

**Description**:
Updates various Russian fireproofing translations to make more sense, see asana task for details

**Steps to test this PR**:
1. Go to settings, and check the fire button animations settings entry, and the title on the page are as specified in the asana task
2. Go to fireproofed websites in settings and do the same. Turn on fireproofing. 
3. Log in to a website so you get the fireproofing dialogue. Check the translations there are correct. Press confirm, and check the toast is correct.



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
